source/team/ways-of-working.html.md.erb

```diff
@@ -24,7 +28,7 @@ As a team we have agreed to do the following for each stage:
     * Tasks are roughly prioritised during a sprint planning session, but we are not obliged to take the one from the top.
     We have the freedom of picking a task that we want to work on and we do not pre-assign team members to a task prior the start of a sprint.
     * Tasks that are selected for a sprint should have all the necessary information in the task description, including a Definition of Done (DoD).
-    If it is not clear what is asked in the task, it does not meet the Definition of Ready (DoR) and should be taken out of the sprint.
+    If it is not clear what is asked in the task, then it does not meet the DoR and therefore should be taken out of the sprint.
```

Comment:
Could you add back in the explanation of what DoR means? I've not recently seen DoR used before, so was slightly confused.

Reply:
it is explained earlier on in the doc, hence I removed it